QuantLib/ql/termstructures/volatility/optionlet/optionletstripper1.cpp

```diff
@@ -24,6 +24,7 @@
 #include <ql/termstructures/volatility/optionlet/optionletstripper1.hpp>
 #include <ql/instruments/makecapfloor.hpp>
 #include <ql/pricingengines/capfloor/blackcapfloorengine.hpp>
+#include <ql/pricingengine.hpp>
```

Comment:
Yes, it is indeed good style, if the indirect inclusion via another file is accidental (thereby creating the possibility for an error if the other file is no longer included or the include chain going from there changes, as in #330, where the situation is even more complicated). But it is different in case of inheritance as here (the Black / Bachelier engines clearly have to include PricingEngine). In any case the redundancy is killed via the include guard, so as I said it is not too important.